CondensateCrew/condensate_backend
# Add Action to Import Words

### What's this PR do?
- Add a words table to the database
- Add a text file of words that can be imported to the database for selection of random words for the game
- Add action to import those words into the database. This means we can append new words to the text file whenever we feel we need to

### Where should the reviewer start?
- `words.txt` in the root folder
- `models.py` has the spec for the Word model
- `app.py` has the `import_nouns` action

### How should this be manually tested?
- Add new words to the text file and visit `/import_noun`, you will see a count of the new words added to the database

Closes #28
